.github/workflows/go.yml

```diff
@@ -0,0 +1,72 @@
+name: Go Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
```

Comment:
Can we add multiple os (os: [ubuntu-latest, macos-latest, windows-latest]) please?
https://github.com/aurghob33/amazon-qldb-driver-java/blob/master/.github/workflows/maven.yml has an example for java.

Reply:
Done.